JSDemos/Demos/DataGrid/Appearance/Angular/app/app.component.html

```diff
@@ -1,9 +1,13 @@
-<dx-data-grid id="gridContainer" #grid
-        [dataSource]="employees"
-        [showColumnLines]="false"
-        [showRowLines]="true"
-        [showBorders]="true"
-        [rowAlternationEnabled]="true">
+<dx-data-grid
+    id="gridContainer"
+    #grid
+    [dataSource]="employees"
+    keyExpr="ID"
+    [showColumnLines]="false"
+    [showRowLines]="true"
+    [showBorders]="true"
+    [rowAlternationEnabled]="true"
+>
```

Comment:
Please fix all entries of this if it can be done automatically.

```suggestion
    [rowAlternationEnabled]="true">
```